src/components/datePicker/DatePicker.tsx

```diff
@@ -80,8 +96,7 @@ export class DatePicker extends React.Component<IDatePickerProps, any> {
 
 
     const inputClasses: string[] = [`border-${this.props.color}`];
-    if (this.props.isSelecting === DateLimits.upper && this.props.upperLimit
-      || this.props.isSelecting === DateLimits.lower && !this.props.upperLimit) {
```

Comment:
I have simply made an attribute out of it (`this.isPicked` initialized in `componentWillReceiveProps`) so it can also be used elsewhere.